src/App.js

```diff
@@ -1,20 +1,46 @@
-import { useContext } from 'react';
+import { useEffect } from 'react';
+import axios from 'axios';
+import {useDispatch, useSelector} from 'react-redux';
 import { Routes, Route, NavLink, Navigate } from 'react-router-dom';
 import Cards from './pages/Cards';
 import PageNotFound from './pages/PageNotFound';
 import SignIn from './pages/SignIn';
```

Comment:
You're using react'18 -> check on using React.lazy(), + Suspence & error boundary components)))